.ci/install_kata_kernel.sh

```diff
@@ -104,6 +122,9 @@ main() {
 		vanilla)
 			build_and_install_kernel "kernel"
 			;;
+		snp)
+			install_kernel "${kernel_type}" $(get_version "assets.kernel.snp.version")
```

Comment:
Since an entry for SNP was added already, I selected that. But they are practically the same. I will change it to SNP since that is what we are doing in main.